### DISCUSSION TITLE
Re-Design Personal Dashboard

### DISCUSSION THREAD

**Discussion by jhdebruin** [djong1/gp3s#217] [Website improvements]
Based on the user survey the Personal Dashboard is the most used feature, it also has not many graphical elements, we are considering some redesign on this part, please let me know your ideas about this, we are open for suggestions and feedback !

![Screenshot 2023-08-31 at 15 08 59](https://github.com/djong1/gp3s/assets/109060128/97c601a9-d4d6-4983-b434-8a05e6c66586)

**Comment by LucSalomons** [comment]
This example looks good! Left bottom graph; is this material used?  I would like this in this page to see used sails and used boards for the year.

**Comment by jhdebruin** [comment]
Would be nice to receive some feedback on this, or do you like it this much we just have to build it :)

**Comment by LucSalomons** [comment]
I guess that current year will be the default for this view. It would nice if one could select an other year and get this summary.

**Comment by djong1** [comment]
are you guy's familiar with this one: https://www.gps-speedsurfing.com/default.aspx?mnu=useryearstats, or should we link to this page for more details and insights?
